example/envoy-tls.yaml

```diff
@@ -0,0 +1,79 @@
+static_resources:
```

Comment:
we could consider a go test framework, that may be used in a simple `go test` command.

in that go test framework, we can start a envoy process with a piece of envoy config, by using Go.

yep, it could be done in a seperate PR.

Reply:
OK. I will do it later.